target/linux/ath79/tiny/base-files/etc/uci-defaults/04_led_migration

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
```

Comment:
please remove the shebang, these files are sourced.

Reply:
Actually, this is a copy-paste leftover from generic subtarget, so it's need to be fixed there also. Will fix here of course :)

Reply:
I'm aware, should be removed there as well. Guess what I compared this with for review ...